upload/admin/view/template/common/column_left.twig

```diff
@@ -5,9 +5,9 @@
     {% for menu in menus %}
       <li id="{{ menu.id }}">
         {% if menu.href %}
-          <a href="{{ menu.href }}"><i class="fas {{ menu.icon }}"></i> {{ menu.name }}</a>
+          <a href="{{ menu.href }}"><i class="{{ menu.icon }}"></i> {{ menu.name }}</a>
         {% else %}
-          <a href="#collapse{{ i }}" data-toggle="collapse" class="parent collapsed"><i class="fa {{ menu.icon }}"></i> {{ menu.name }}</a>
+          <a href="#collapse{{ i }}" data-toggle="collapse" class="parent collapsed"><i class="{{ menu.icon }}"></i> {{ menu.name }}</a>
```

Comment:
I don't see any fa class on that line ... I do see fas like line 8 though.

Reply:
FontAwesome5 have no fa class. So this was a typo, left from old OC versions. 
https://fontawesome.com/how-to-use/on-the-web/referencing-icons/basic-use

My proposal is to pass icon classes in column_left.php. So that in the future it is possible to use other FontAwesome5 classes like far, fab and others which are available in free version. 
https://github.com/opencart/opencart/pull/8937

These icons can be used when adding new menu branches for extensions using events system. Otherwise all icons are limited only with fas class, which is wrong and will not allow to use hundreds of icons from other classes.